internal/manifest/referencevalues.go

```diff
@@ -0,0 +1,100 @@
+// Copyright 2024 Edgeless Systems GmbH
+// SPDX-License-Identifier: AGPL-3.0-only
+
+package manifest
+
+import (
+	_ "embed"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"strconv"
+)
+
+var (
+	// EmbeddedReferenceValuesJSON contains the embedded reference values in JSON format.
+	// At startup, they are unmarshaled into a globally-shared ReferenceValues struct.
+	//
+	//go:embed assets/reference-values.json
+	EmbeddedReferenceValuesJSON []byte
+
+	embeddedReferenceValues *ReferenceValues
```

Comment:
Why do we need this global state?

Reply:
To not unmarshal it over and over?

Reply:
How often would we need to unmarshal it? What is the cost?

Reply:
Every time someone needs the reference values. Which happens every time the runtime handler is required, at least. So at generation time (with the current code - for each generation of a resource), thus also in each e2e test

I just don't see any value handling a static value as if it were dynamic